vendor/k8s.io/kubernetes/staging/src/k8s.io/apiserver/pkg/storage/etcd/testing/utils.go

```diff
@@ -188,7 +188,7 @@ func configureTestCluster(t *testing.T, name string, https bool) *EtcdTestServer
 // launch will attempt to start the etcd server
 func (m *EtcdTestServer) launch(t *testing.T) error {
 	var err error
-	if m.s, err = etcdserver.NewServer(&m.ServerConfig); err != nil {
+	if m.s, err = etcdserver.NewServer(m.ServerConfig); err != nil {
```

Comment:
this makes me sad... we *just* got to the point where openshift/kuberenetes could be built/run

Reply:
@liggitt do you know how far is upstream with bumping the etcd?

Reply:
looks like plan is for 3.3 in 1.11: https://github.com/kubernetes/kubernetes/issues/61326#issuecomment-374737257

Reply:
@ingvagabund @liggitt in that case, I would prefer to wait for 3.11 with bumping the etcd client, so we can get upstream to update the deps and the config structs, which will mean we won't risk nasty regressions and we will also be able to build the 1.10 openshift/kubernetes.